src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,32 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class Line {
+
+  private final List<Point> points = new ArrayList<>();
+
+  public Line(int countOfPerson) {
+    IntStream.range(0, countOfPerson)
+        .forEach(i -> points.add(connectPoint(new RandomConnect())));
+  }
+
+  public Point connectPoint(ConnectStrategy connectStrategy) {
+    if (points.size() == 0) {
```

Comment:
```suggestion
    if (points.isEmpty()) {
```

List 인터페이스에서 기본적으로 제공하는 메서드를 활용하면 어떨까 해서 제안해보았습니다.